Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -743,6 +746,24 @@ void MainWidget::create_default_image()
     editor.set_active_layer(bg_layer);
 }
 
+bool MainWidget::request_close()
+{
+    if (m_tab_widget->children().size() == 0)
```

Comment:
nit: We should use `AK::Vector::is_empty()`
```suggestion
    if (m_tab_widget->children().is_empty())
```